Week-5/asyncOp.js

```diff
@@ -0,0 +1,71 @@
+const getNumber1 = () =>
```

Comment:
one function with async await and other function using generator

Reply:
made changes as discussed